test/FastTests/Sparrow/AdvMemoryTests.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Runtime.Intrinsics.X86;
+using Xunit;
+using Xunit.Abstractions;
+using Sparrow;
+
+namespace FastTests.Sparrow
+{
+    public unsafe class AdvMemoryTests(ITestOutputHelper output) : NoDisposalNeeded(output)
+    {
+        [Fact]
```

Comment:
For new tests please use `RavenFact` - otherwise `SlowTests.Tests.TestsInheritanceTests.AllTestsShouldUseRavenFactOrRavenTheoryAttributes` will be failing

Reply:
Done